src/main.c

```diff
@@ -716,7 +716,9 @@ uint32_t sign_hash_and_set_result(void)
             rng_rfc6979(G_io_apdu_buffer + 100, tmpCtx.transactionContext.hash, NULL, 0, SECP256K1_N, 32, V, K);
         }
         uint32_t infos;
-        tx = cx_ecdsa_sign(&privateKey, CX_NO_CANONICAL | CX_RND_PROVIDED | CX_LAST, CX_SHA256,
+        // ignores returns signature length which is always 64, we always return length of 65
```

Comment:
I don't see a return statement in this function. It looks like we actually return garbage!

Also, this function seems to potentially be non-returning? Is that correct behavior?

We likely want to do one of the following:
 - Retain the capture of the value for `tx` and add one to it in the return statement.
 - Remove `tx` entirely and simply return `65`.